File: src/main/java/dev/emortal/minestom/marathon/util/NoPhysicsEntity.java
package dev.emortal.minestom.marathon.util;

import net.minestom.server.entity.Entity;
import net.minestom.server.entity.EntityType;

public class NoPhysicsEntity extends Entity {

    public NoPhysicsEntity(EntityType entityType) {
        super(entityType);
        hasPhysics = false;
    }
}

File: src/main/java/dev/emortal/minestom/marathon/util/NoPhysicsEntity.kt
package dev.emortal.minestom.marathon.util

import net.minestom.server.entity.Entity
import net.minestom.server.entity.EntityType

class NoPhysicsEntity(entityType: EntityType?) : Entity(
    entityType!!
) {
    init {
        hasPhysics = false
    }
}
